angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead-selector/sprk-masthead-selector.component.spec.ts

```diff
@@ -0,0 +1,400 @@
+import { Component } from '@angular/core';
+import { RouterTestingModule } from '@angular/router/testing';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+
+import { SprkIconComponent } from '../../sprk-icon/sprk-icon.component';
+import { SprkLinkDirective } from '../../../directives/sprk-link/sprk-link.directive';
+import { SprkMastheadSelectorComponent } from './sprk-masthead-selector.component';
+import { ISprkMastheadSelectorChoice } from './sprk-masthead-selector.interfaces';
+
+@Component({
+  template: `
+    <sprk-masthead-selector
```

Comment:
Remove tests that correspond to the extra inputs that will be removed in the component.